FunctionalPreprocessing/FunctionalPreprocessing.XNAT_MARK_RUNNING_STATUS

```diff
@@ -77,14 +93,22 @@ get_options()
 				g_subject=${argument#*=}
 				index=$(( index + 1 ))
 				;;
-			--classifier=*)
-				g_classifier=${argument#*=}
-				index=$(( index + 1 ))
-				;;
+            --session=*)
```

Comment:
I prefer that we not pass in `--session=<value>` and stick with the approach that was previously used in this script of passing in the classifier, `--classifier=<value>`. We can build the session value by concatenating together the `${g_subject}` value and the `${g_classifier}` value with an underscore, `_`, between them.

I know this isn't consistent with teh way `PutDirIntoResource.sh` and `RemoveFileFromResource.sh` are written. They expect a `--session=<value>` parameter. But I've learned since writing that code that the classifier is a better piece of information to pass around.

Reply:
updated to classifier